tools/update-gcp-infra.sh

```diff
@@ -0,0 +1,168 @@
+#!/bin/bash
+# Turbinia GCP management script
+# This script can be used to manage a Turbinia stack deployed to GCP. The only requirement is you have 'gcloud' installed.
+# Use --help to show you commands supported.
+
+set -o posix
+
+CMD=$1 
+INSTANCEID=$2
+ZONE=$3
+DOCKER_TAG=$4
+CONFIG_FILE=$4
+
+SERVER_URI="us-docker.pkg.dev/osdfir-registry/turbinia/release/turbinia-server"
+WORKER_URI="us-docker.pkg.dev/osdfir-registry/turbinia/release/turbinia-worker"
+GCLOUD=`command -v gcloud`
+
+function usage { 
+  echo "Usage: $0 COMMAND INSTANCEID ZONE [DOCKER_TAG or CONFIG_FILE]" 
+  echo "COMMAND         Pick one of the commands below"
+  echo "INSTANCEID      The Turbinia deployment instance ID"
+  echo "ZONE            The GCP zone the Turbinia deployment is located"
+  echo
+  echo "Optional arguments:"
+  echo "DOCKER_TAG      Docker image tag, eg latest or 20210606"
+  echo "CONFIG_FILE     Path to  Turbinia configuration file"
+  echo 
+  echo "Commands supported:"
+  echo "change-image    Change the docker image loaded by a Turbinia deployment with DOCKER_TAG"
+  echo "logs            Display logs of a Turbinia server or worker"
+  echo "show-config     Write the Turbinia configuration of an instance to STDOUT"
+  echo "status          Show the running status of server and workers"
+  echo "start           Start a Turbinia deployment"
+  echo "stop            Stop a Turbinia deployment"
+  echo "update-config   Update the Turbinia configuration of a Turbinia deployment from CONFIG_FILE"
+  echo
+
+  exit 1
+}
+
+function check_gcloud {
+    if [ -z $GCLOUD ]
+    then
+        echo "gcloud not found, please install first"
+        exit 1
+    fi
+}
+
+function show_infra {
+    $GCLOUD compute instances list --filter="zone:$ZONE" --filter="name ~ $INSTANCEID"
```

Comment:
The first command that is executed is the '$GCLOUD config list project' command. This will show you the configured project or tell you to set it if it is not set (through authentication or gcloud config set project).